package/snapper.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon May 25 11:41:59 UTC 2015 - jreidinger@suse.com
+
+- allow list-configs command to work on different root
+  (fate#317900)
```

Comment:
I usually write "for fate ..." since the improvements here is only accessory part.